manifests/emissary/emissary-defaultns.yaml.in

```diff
@@ -626,9 +626,7 @@ spec:
         product: aes
     spec:
       containers:
-      - command:
```

Comment:
I assume this is being removed because the new container already calls it via CMD/ENTRYPOINT on startup?

Reply:
yup